### DISCUSSION TITLE
Recent changes to counters of changed lines

### DISCUSSION THREAD

**Discussion by elias6** [community/community#54471] [Pull Requests]
### Select Topic Area

Bug

### Body

There have been 2 recent changes to pull requests in the last few weeks, which I do _not_ like:
1. Before, when viewing a PR with the "hide whitespace" setting enabled, the red and green indicator showing the number of lines changed did not include lines with only whitespace changes. Now, it shows the total number of changed lines (as if "hide whitespace" were disabled).
2. Before, if I hovered my mouse over the red and green indicator for an individual file, it showed the number of added and removed lines. Now, it doesn't show anything.

I don't know if these are bugs or deliberate changes. But I don't see any reason for them. I miss the old behavior.

**Comment by corbinschwartz** [comment]
This has been frustrating me as well, is this issue on anyone's radar at GitHub?
